htdocs/adherents/list.php

```diff
@@ -1428,7 +1428,7 @@
 
 print '</form>'."\n";
 
-if (in_array('builddoc', $arrayofmassactions) && ($nbtotalofrecords === '' || $nbtotalofrecords)) {
+if (in_array('builddoc', array_map('trim', array_keys($arrayofmassactions))) && ($nbtotalofrecords === '' || $nbtotalofrecords)) {
```

Comment:
Why this ?
arrayofmassactions is an array filled by code. If there is an error into content, we must fix when we fill it at the source of the trouble and not later.